app/views/vacancies/index.html.haml

```diff
@@ -23,19 +23,20 @@
             = render partial: 'vacancy', locals: { vacancy: vacancy }
 
       - elsif @vacancies.user_search?
-        .divider-bottom
-          %p.govuk-heading-m
-            = t('jobs.no_jobs.intro')
+        #server-no-results
```

Comment:
You can just add this id to the other element rather than nesting it in another div element - unless that was the intention?